src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,78 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { filterTodo } from '../../features/filterTodo';
+import { actions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const [currentUser, setCurrentUsers] = useState<User>();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const todos = useAppSelector(filterTodo.getFilteredTodos);
+  const dispatch = useAppDispatch();
+
+  const removeCurrentTodo = () => {
+    dispatch(actions.removeTodo());
+  };
+
+  useMemo(() => {
```

Comment:
You cannot directly use the `useMemo` hook without assigning its return value to a variable. The `useMemo` hook is designed to be used with variable assignment so that the memoized value can be stored and reused.